unix-alpha/setup.py

```diff
@@ -1,11 +1,11 @@
 from distutils.core import setup
 
-with open("README.md", "r") as fh:
+with open("alpha-description.md", "r") as fh:
     long_description = fh.read()
 
 setup(
     name="machine_stats_alpha",
-    version="1.0.0",
+    version="develop",
```

Comment:
How would the increment work here @pranavmalaviya?  Is the strategy different?

Reply:
Same as PR #210, the increment will be done from tags and we will replace this 👆 line using a `sed` command.